client/components/DataManagement/DataManagementRow/index.jsx

```diff
@@ -541,14 +541,16 @@ const DataManagementRow = ({
                 // "active"
                 insertActivePhaseForTestPlan(latestVersion);
                 return (
-                    <PhaseCell>
+                    <PhaseCell role="list">
                         <VersionString
+                            role="listitem"
```

Comment:
In the VersionString component, you should take the rest props `...props` and then insert them in the first child, or else the VersionString will be missing the listitem role.

Reply:
Good catch, and I also did the same with `TestPlanReportStatusDialogWithButton`